loaddictionaries.cc

```diff
@@ -117,7 +117,7 @@ void LoadDictionaries::handlePath( Config::Path const & path )
   for( QFileInfoList::const_iterator i = entries.constBegin();
        i != entries.constEnd(); ++i )
   {
-    QString fullName = i->canonicalFilePath();
+    QString fullName = i->absolutefilePath();
```

Comment:
I think you have a typo (wrong case) here. This probably doesn't compile.

Reply:
and please adjust the commit message as well.

Reply:
Sorry, my bad. Also change `CanonicalPath` to `canonicalFilePath`, stick to qt API.
Thanks for strict code reviewing.